server/bundler/loader.js

```diff
@@ -86,7 +86,7 @@ function splitTemplate( path, module, chunkName ) {
 		'		}',
 		'		context.store.dispatch( { type: "SET_SECTION", isLoading: false } );',
 		'		if ( ! _loadedSections[ ' + JSON.stringify( module ) + ' ] ) {',
-		'			require( ' + JSON.stringify( module ) + ' )( controller.clientRouter );',
+		'			require( ' + JSON.stringify( module ) + ' )( controller.clientRouter, controller.clientRenderer );',
```

Comment:
Naive question: what happens here for controllers that don't have these functions?